### PR TITLE
Reorganize install.rst page

### PR DESCRIPTION
**Pull request by registerrier**
**Description**

This pull request moves dataset download to the install.rst page following the suggestion made during the dev call. 
- The page is split into 4 sections: `Using conda`, `Using other package managers`, `Download tutorials` and a short `What next?`.
- The paragraph mentioning download of different parts of the data/tutorials has been removed, since it has been decided to move to a single tar file download.
- The next button now links to the getting started page, which no longer includes the download part.
- The examples in the getting started page have been updated. 

**Dear reviewer**

Is this OK @Bultako @adonath @AtreyeeS ?

### CODE REVIEW

File: docs/install/index.rst [hunk 1]
```diff
@@ -6,26 +6,30 @@
...
-This creates a conda environment called ``gammapy-0.16`` which you can activate via:
+This creates a conda environment called ``gammapy-0.17`` which you can activate via:
 
 .. code-block:: bash
 
-    conda activate gammapy-0.16
+    conda activate gammapy-0.17
 
 Note that you have to execute that activation command (but not the environment
 cration command) every time you open a new terminal window, because the default
```
Comment (AtreyeeS) [inline docs/install/index.rst]: creation

File: docs/install/index.rst [hunk 2]
```diff
@@ -37,17 +41,54 @@ To check your Gammapy installation, you can use this command:
...
     gammapy info
 
-Congratulations! You are all set to start using Gammapy!
 
-If you're new to Python, ipython and Jupyter, read the :ref:`getting-started`
-guide. To learn how to use Gammapy, go to :ref:`tutorials`.
+Using other package managers
+----------------------------
 
 The following pages contain detailed information about Gammapy dependencies in
```
Comment (AtreyeeS) [inline docs/install/index.rst]: Maybe mention something like "If you don't want to use Anaconda, you can...".
This makes it clearer to python novices that once the installation is successful using conda, there is no requirement to go through this sub-section or bother about dependencies explicitly.

File: docs/install/index.rst [hunk 3]
```diff
@@ -37,17 +41,54 @@ To check your Gammapy installation, you can use this command:
...
+
+.. code-block:: bash
+
+    gammapy download tutorials --release 0.17
+    cd gammapy-tutorials
+    export GAMMAPY_DATA=$PWD/datasets
+
+You might want to put the definition of the ``$GAMMAPY_DATA`` environment
+variable in your shell profile setup file that is executed when you open a new
+terminal (for example ``$HOME/.bash_profile``).
```
Comment (AtreyeeS) [inline docs/install/index.rst]: I think in linux machines it's `$HOME/.bashrc` ?

File: docs/install/index.rst (hunk 1 above)
Comment (registerrier) [reply docs/install/index.rst]: Done

File: docs/install/index.rst (hunk 2 above)
Comment (registerrier) [reply docs/install/index.rst]: Good point. Added

File: docs/install/index.rst (hunk 3 above)
Comment (registerrier) [reply docs/install/index.rst]: Technically, when you open a new terminal, it is probably `$HOME/.bashrc` that is executed.

File: docs/install/index.rst [hunk 4]
```diff
@@ -37,17 +41,54 @@ To check your Gammapy installation, you can use this command:
...
-various installation options:
 
-.. toctree::
-    :maxdepth: 1
+Download tutorials
+------------------
+
+You can now proceed to download the Gammapy tutorial notebooks and the example
+datasets used there (at the moment from CTA, H.E.S.S. and Fermi-LAT). The total
+size to download is about 100 MB.
```
Comment (Bultako) [inline docs/install/index.rst]: The total size of the datasets folder when uncompressed is actually about 180 MB.

File: docs/install/index.rst [hunk 5]
```diff
@@ -37,17 +41,54 @@ To check your Gammapy installation, you can use this command:
...
+
+.. code-block:: bash
+
+    gammapy download tutorials --release 0.17
+    cd gammapy-tutorials
+    export GAMMAPY_DATA=$PWD/datasets
+
+You might want to put the definition of the ``$GAMMAPY_DATA`` environment
```
Comment (Bultako) [inline docs/install/index.rst]: You might want to move the `datasets` folder anywhere else in your hard disk, and then put the definition of the ``$GAMMAPY_DATA`` environment variable pointing to this folder, in your shell profile setup file that is executed when you open a new terminal (for example ``$HOME/.bash_profile``).

File: docs/install/index.rst [hunk 6]
```diff
@@ -37,17 +41,54 @@ To check your Gammapy installation, you can use this command:
...
+
+What next?
+----------
+
+Congratulations! You are all set to start using Gammapy!
 
-    dependencies
-    pip
-    other
+* If you're new to Python, ipython and Jupyter, read the :ref:`getting-started` guide.
```
Comment (Bultako) [inline docs/install/index.rst]: If you're new to conda, Python, ipython and Jupyter

File: docs/install/index.rst (hunk 4 above)
Comment (registerrier) [reply docs/install/index.rst]: Changed. Thanks.

File: docs/install/index.rst (hunk 6 above)
Comment (registerrier) [reply docs/install/index.rst]: Done

File: docs/install/index.rst (hunk 5 above)
Comment (registerrier) [reply docs/install/index.rst]: I slightly adapted following the line you suggest.

**Comment by registerrier** [comment]
Suggested changes have been included. Travis issues are unrelated. Merging now.

**Pull request merged**